src/commands/installExtension.ts

```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import { commands } from "vscode";
+
+export async function installExtension(_context: IActionContext, extensionId: string): Promise<void> {
+    void commands.executeCommand('extension.open', extensionId);
+    void commands.executeCommand('workbench.extensions.installExtension', extensionId);
```

Comment:
It would already be happening via the `callWithTelemetryAndErrorHandling` implicitly wrapped around any command registered with `registerCommand`, so I think we're good.

Reply:
Yeah, I was thinking we wouldn't want to await either command, but I think it's okay to await the installExtension prompt.